src/common/uilocale.cpp

```diff
@@ -752,9 +752,14 @@ const wxLanguageInfo* wxUILocale::FindLanguageInfo(const wxString& locale)
     // to the entry description in the language database.
     // The locale string may have the form "language[_region][.codeset]".
     // We ignore the "codeset" part here.
+    wxString myLocale = locale;
+    if (myLocale.IsSameAs("C", false) || myLocale.IsSameAs("POSIX", false))
```

Comment:
Another minor comment, but I'd add a helper function to use it here and in the other place where we perform the same test to allow writing just
```suggestion
    if (IsDefaultCLocale(myLocale))
```